resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -308,3 +308,7 @@ aws.settings.executables.find.description=Select path to {0} executable
 aws.settings.executables.cannot_determine_version=Cannot determine version of {0}: {1}
 aws.settings.executables.executable_invalid=Invalid {0} executable: {1}
 aws.settings.executables.resolution_exception=Exception occurred attempting to resolve {0} executable: {1}
+
+notice.suppress=Don't show this notification again
+notice.title.jetbrains.minimum.version.2019.2=AWS Toolkit Requirements Notice
+notice.message.jetbrains.minimum.version.2019.2=An upcoming release of the AWS Toolkit will require version 2019.2 or newer of the JetBrains IDE.
```

Comment:
Can we get smart about the IDE name? I think it would be clearer to customers if we said something like "require version 2019.2 of IntelliJ" /PyCharm/etc; should be able to use `ApplicationNamesInfo.getInstance().fullProductName`

Reply:
I like this suggestion, I'm taking a look

Reply:
Coming soon:
![image](https://user-images.githubusercontent.com/39839589/66092179-d2921480-e53f-11e9-9d8c-ab7d966eb0a2.png)

Reply:
![image](https://user-images.githubusercontent.com/39839589/66092246-23a20880-e540-11e9-9ade-3afcda7bac99.png)